docs/content/docs/data-structures/counter.md

```diff
@@ -1,5 +1,146 @@
 ---
 title: SharedCounter
-menuPosition: 9
-status: unwritten
+menuPosition: 5
 ---
+
+## Introduction
+
+The `SharedCounter` distributed data structure (DDS) is use to store an integer counter value that can be modified collaboratively.
+The data structure affords incrementing and decrementing the shared value via its `increment` method (decrements are handled by providing a negative delta value).
+
+The `SharedCounter` is a specialized, [Optimistic DDS]({{< relref "dds.md#optimistic-data-structures" >}}).
+It operates on communicated deltas, rather than direct changes to the shared value.
+In this way, it avoids the pitfalls of DDSes with simpler merge strategies, in which one user's edit may clobber another's.
+
+Note that the `SharedCounter` only operates on integer values.
+Floating point arithmatic is order-dependent.
```

Comment:
That's fair. My thinking was that since the API operations on `number`s at the type level, it would be nice to explain why we further restrict the input space at runtime. This article probably doesn't need that note, but I think the type itself should probably call this out in a `@remarks` block if it doesn't already.

Reply:
And yes, we should totally have trivia-style callouts. I love the idea of our tutorial docs looking a lot like less-formal textbooks.